code/style.css

```diff
@@ -2,6 +2,175 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
-.active + .answer {
-  /* some styling here */
+/* Hero Section */
+
+body {
+  background-image: url("./images/garden.jpg");
+  margin:0px;
+  object-fit: cover;
+  background-attachment: fixed;
+}
+
+.hero-header {
+  position:relative;
+
+}
+.hero-image {
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+  border-bottom: solid;
+}
+
+.tenko {
+  position: absolute;
+  top:0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background-color: rgba(255, 255, 0, 0.5);
+  transform: rotate(90deg);
```

Comment:
Looks like this is making your content to wide (i.e. not responsive):
<img width="417" alt="Skärmavbild 2021-02-04 kl  18 14 32" src="https://user-images.githubusercontent.com/51529201/106929748-09b89d00-6715-11eb-92e3-02489eb51140.png">